packages/config/src/validate/validations.js

```diff
@@ -149,6 +156,22 @@ const POST_NORMALIZE_VALIDATIONS = [
     ...insideRootCheck,
     example: (edgeHandlers) => ({ build: { edge_handlers: removeParentDots(edgeHandlers) } }),
   },
+  {
```

Comment:
Should we add a "pre-normalize" validation to ensure `build.functions` is a plain object? 
Otherwise if (for example) the user is using a string, `@netlify/config` will crash without a proper validation message.

Reply:
Yes! Done in https://github.com/netlify/build/pull/2390/commits/30ae3eb21fd75eb62826131fcf283e6fdd261ff4.